system/Session/Handlers/DatabaseHandler.php

```diff
@@ -282,12 +282,9 @@ public function destroy($id): bool
     #[ReturnTypeWillChange]
     public function gc($max_lifetime)
     {
-        $separator = ' ';
-        $interval  = implode($separator, ['', "{$max_lifetime} second", '']);
-
         return $this->db->table($this->table)->where(
             'timestamp <',
-            "now() - INTERVAL {$interval}",
+            "now() - INTERVAL  {$max_lifetime} second ",
```

Comment:
```suggestion
            "now() - INTERVAL {$max_lifetime} second",
```

Reply:
I don't have a local GPG key so I won't be able to make a secure commit. Feel free to close this pull request and make the change to the repo directly I have no interest in contribution recognition.

Reply:
There is an option to approve the suggestion, which I used.